src/stylesheets/colors.scss

```diff
@@ -1,3 +1,5 @@
+$focuscolor: #0F465C;
```

Comment:
The entire `opds-web-client` package was deleted but this and the `basic-focus` mixin had to be moved into this repo.